File: uva_11526.cpp
# include <iostream>
# include <cmath>
using namespace std;
long long H(int n){
    long long ans=0, tmp=sqrt(n);
    for(int i=1; i<=tmp;i++)
        ans+=(n/i);
    return 2*ans-tmp*tmp;
}
int main(void){
    int n, num;
    cin >> num;
    while(num--){
        cin >> n;
        cout << H(n) << endl;
    }
    return 0;
}

File: uva_11526.py
import math
def H(n):
    if n<=0:
        return 0
    ans, i=0, 1
    tmp=int(math.sqrt(n))
    while i<=tmp:
        ans+=int(n/i)
        i+=1
    return 2*ans-tmp*tmp
num=int(input())
while num:
    num-=1
    n=int(input())
    print(H(n))